src/data-workspace/data-value-mutations/index.js

```diff
@@ -1,7 +1,9 @@
 export {
-    getDataValueMutationKey,
     useSetDataValueMutation,
     useDeleteDataValueMutation,
     useUploadFileDataValueMutation,
+    useSetDataValueMutationDefaults,
 } from './data-value-mutations.js'
 export { useDataValueParams } from './use-data-value-params.js'
+export * from './mutation-functions.js'
```

Comment:
I don't see any of these mutation functions imported from other files -- is this export needed?

Reply:
This one is used in `query-client` to set the defaults. I didn't touch the other ones above.

Reply:
Ah, true. Sorry I must've read this wrong. Yeah, I'm sure this can be removed.